src/components/client/landing/latestNotice/LatestNotice.jsx

```diff
@@ -70,40 +70,47 @@ function LatestNotice() {
       <div className="slideshow-container">
         <h2>LATEST NOTICES </h2>
       </div>
-      <div className="carousel">
-        <div className="carousel_track-container">
-          <ul className="carousel__track">
-            {images.length > 0 ? (
-              images.map((image, index) => (
-                <li key={index} className="carousel_slide current-slide">
-                  <img
-                    className="carousel-images"
-                    src={image}
-                    alt="notice"
-                    onError={(e) => {
-                      e.target.onerror = null
-                      e.target.src = require('../../../../assets/images/notice/error.webp')
-                    }}
-                  />
-                </li>
-              ))
-            ) : (
-              <li className="carousel_slide current-slide">
-                <div className="carousel-blank-slide">Latest Notices</div>
-              </li>
-            )}
-          </ul>
+      <div className="container-fluid">
+        <div className="row">
+          <div className="carousel">
+            <div className="carousel_track-container">
+              <ul className="carousel__track">
+                {images.length > 0 ? (
+                  images.map((image, index) => (
+                    <li key={index} className="carousel_slide current-slide">
```

Comment:
Since images is an array of images, this doesn't have id attributes. In this case, index should be used. Btw, thank you for that article. I didn't know the cons of using index